src/main/resources/api.yaml

```diff
@@ -360,6 +407,19 @@ definitions:
       - timeAvailable
       - timeRemaining
 
+  UserDto:
```

Comment:
To jest/będzie też u @kjarosh (podobnie jak `UserRoleDto`) - ostrzegam, że ktoś będzie rozwiązywał konflikty w pewnym momencie